tests/Front/ValidPrograms/56_ArrayAsParam.c

```diff
@@ -1,21 +1,13 @@
 #include <stdint.h>
 
 int32_t function(int32_t a[], int32_t n) {
-   int32_t i;
-   int32_t result;
-   result = 0;
-   for (i=0;i<n;i=i+1) {
-      result = result+a[i];
-   }
+   int32_t i = a[i];
```

Comment:
Normalement on devrait pas modifier les tests donnés par le prof, mais plutôt en ajouter dans `Custom`

Reply:
Yep mais ce test ne pouvait pas fonctionner avec la grammaire demandée

Reply:
🤔 Mouais

Reply:
il y avait un "for" dedans.

Reply:
i kno